code42.json

```diff
@@ -1924,6 +1924,964 @@
                 "type": "table"
             },
             "versions": "EQ(*)"
+        },
+        {
+            "action": "hunt file",
+            "identifier": "hunt_file",
+            "description": "Searches Code42 for a backed-up file with a matching hash and downloads it",
+            "verbose": "",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "hash": {
+                    "description": "The md5 or sha256 hash of the file to download",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "md5",
```

Comment:
I didn't use `hash` because some decompiled stuff that I saw suggested that that would make this allowed to be attempted on other types of hashes that we don't support like `SHA1`.